tests/functional/workloads/cnv/test_multi_vm_configurations.py

```diff
@@ -0,0 +1,128 @@
+        (
+            self.vm_objs_def,
+            self.vm_objs_aggr,
+            self.sc_obj_def_compr,
+            self.sc_obj_aggressive,
+        ) = multi_cnv_workload(namespace=proj_obj.namespace)
+
+        logger.info("All vms created successfully")
+
+    def verify_keyrotation(self, vm_objs, sc_obj):
+        for vm in vm_objs:
+            if vm.volume_interface == constants.VM_VOLUME_PVC:
+                pvk_obj = PVKeyrotation(sc_obj)
+                volume_name = vm.pvc_obj.get().get("spec", {}).get("volumeName")
+                volume_handle = None
+                for line in run_oc_command(
+                    f"describe pv {volume_name}", namespace=vm.namespace
+                ):
+                    if "VolumeHandle:" in line:
+                        volume_handle = line.split()[1]
+                        break
+                if not volume_handle:
+                    logger.error(f"Cannot get volume handle for pv {volume_name}")
+                    raise Exception("Cannot get volume handle")
+                assert pvk_obj.wait_till_keyrotation(
+                    volume_handle
+                ), f"Failed to rotate Key for the PVC {vm.pvc_obj.name}"
+
+    @workloads
+    @pytest.mark.polarion_id("OCS-6298")
+    def test_cnv_vms(self, setup):
+        """
+        Tests to verify configuration for non-GS like environment
+
+        Steps:
+        1) Create VMs using fixture multi_cnv_workload
+        2) Validate data integrity using md5sum.
+            a. create file locally and take md5sum
+            b. copy same file to vm and take md5sum
+            c. Validate both are same or not
+        3) Validate pvc level key rotation
+        4) Stop the VM
+        5) Delete the VM (as part of factory teardown)
+
+        """
+
+        # To Do
+        # 1. if os is windows then check rxbounce enabled in sc yaml
+
+        all_vm_list = self.vm_objs_def + self.vm_objs_aggr
+
+        # 2.Validate data integrity using md5sum.
+        file_name = "/tmp/dd_file"
+        vm_filepath = "/home/admin/dd_file1_copy"
+
+        # Create file locally
+        cmd = f"dd if=/dev/zero of={file_name} bs=1M count=1024"
+        run_cmd(cmd)
+
+        # Calculate the MD5 checksum
+        if file_name:
+            cmd = f"md5sum {file_name}"
+            md5sum_on_local = run_cmd(cmd).split()[0]
+            if md5sum_on_local:
+                logger.info(f"MD5 checksum of the file: {md5sum_on_local}")
+            else:
+                raise ValueError(
+                    "MD5 checksum could not be calculated. Ensure the file exists and is accessible."
+                )
+        else:
+            raise ValueError(
+                "File name is not provided. Please specify a valid file name."
+            )
+
+        # Copy local file to all vms
+        for vm_obj in all_vm_list:
+            vm_obj.scp_to_vm(
+                local_path=file_name,
+                vm_dest_path=vm_filepath,
+            )
+
```

Comment:
you can delete these empty lines here and all the other places in this test function